frontend/src/services/authService.js

```diff
@@ -0,0 +1,27 @@
+
+import { env } from '../env';
+
+const isAuthenticated = () => {
+
+    return new Promise( resolve => {
+        fetch(`http://localhost:${env.PORT}/api/auth`, {
```

Comment:
@gulyaasjaanos the whole url of the backend should be in the env, the localhost and even the port could be different
```suggestion
        fetch(`${env.BACKEND_URL}/api/auth`, {
```